src/app/components/NavBar/NavBar.module.css

```diff
@@ -0,0 +1,18 @@
+.navBar {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  background-color: var(--secondary-text-color);
+  padding: 0.5em 0;
+  position: relative;
+  width: 500px;
```

Comment:
The fixed width is needed for the drop placement?

Reply:
Yes, but only for the moment until I find a solution to adapt it to the viewport.